docs/src/pages/roadmap.tsx

```diff
@@ -78,12 +78,14 @@ import {
   mdiLinkEdit,
   mdiTagFaces,
   mdiMovieOpenPlayOutline,
+  mdiCast,
 } from '@mdi/js';
 import Layout from '@theme/Layout';
 import React from 'react';
 import { Item, Timeline } from '../components/timeline';
 
 const releases = {
+  'v1.133.0': new Date(2025, 4, 21),
```

Comment:
May need to update, depending on when we release

Reply:
No, months in JS Dates are 0 based. It's kind of weird https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_components_and_time_zones

Reply:
Oh interesting, sorry to bother!

Reply:
https://immich.app/cursed-knowledge/ :P

Reply:
<img width="733" alt="Screenshot 2025-05-21 at 20 52 57" src="https://github.com/user-attachments/assets/060857cb-6c32-4ba0-a889-a660d25a5cf4" />
Yup :D